### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by nialloulton**
Updating the website with MMM-GPT information and link

----
:books: Documentation preview :books:: https://pymc-marketing--433.org.readthedocs.build/en/433/

### CODE REVIEW

**Comment by nialloulton** [comment]
@twiecki

File: docs/source/index.md [hunk 1]
```diff
@@ -60,6 +60,13 @@ To the reference guide
 ::::
 :::::
 
+## Marketing AI Assistant: MMM-GPT with PyMC-Marketing
```
Comment (twiecki) [inline docs/source/index.md:63]: Let's move it further down.

File: docs/source/index.md [hunk 2]
```diff
@@ -60,6 +60,13 @@ To the reference guide
...
+
+**[Try MMM-GPT Now](https://mmm-gpt.com/)**
+
+#### About MMM-GPT
```
Comment (twiecki) [inline docs/source/index.md:67]:
```suggestion
### About MMM-GPT
```

**Comment by twiecki** [comment]
Let's reuse the text from https://github.com/pymc-labs/pymc-marketing/pull/434.

**Comment by ricardoV94** [comment]
@nialloulton @twiecki is this one still relevant?

**Comment by juanitorduz** [comment]
@nialloulton I think we can close this one as it seems a duplicate by https://github.com/pymc-labs/pymc-marketing/pull/434. we can always re-open it :) Is just we are doing some backlog cleaning 🙏

**Pull request closed without merging**